spec/models/user_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  it 'Testing user without name' do
+    user = User.create
+    expect(user).to_not be_valid
+  end
+
+  it 'Testing user PostCounter number negative' do
+    user = User.create(Name: 'Tanusri', Image: 'https://unsplash.com/photos/F_-0BxGuVvo', Bio: 'Microverse')
+    user.PostCounter = -10
+    expect(user).to_not be_valid
+  end
+
+  it 'Name should be present' do
+    user = User.create(Name: 'Tanusri', Image: 'https://unsplash.com/photos/F_-0BxGuVvo', Bio: 'Microverse')
+    user.Name = nil
+    expect(user).to_not be_valid
+  end
+
+  it 'Has a latest_posts method' do
```

Comment:
- [ ] As the previous reviewer mentioned kindly make sure to fix these errors so the rspec tests are passing 👇🏼 
![Screenshot_20230128_204722](https://user-images.githubusercontent.com/25227022/215287904-4f6065f6-82b9-454c-9a52-f04e521cf285.png)